.github/workflows/ci.yml

```
@@ -0,0 +1,368 @@
+    if: ${{ github.event_name == 'pull_request' && needs.detect-changes.outputs.php_deps == 'true' }}
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v6
+
+      - name: Setup PHP
+        uses: shivammathur/setup-php@v2
+        with:
+          php-version: '8.2'
+          tools: composer
+          cache: composer
```

actionlint (1.7.10)

input "cache" is not defined in action "shivammathur/setup-php@v2". available inputs are "coverage", "extensions", "github-token", "ini-file", "ini-values", "php-version", "php-version-file", "tools"

(action)